src/schemas/21-526EZ-allclaims/schema.js

```diff
@@ -646,7 +646,18 @@ const schema = {
                   type: 'object',
                   properties: {
                     name: {
-                      $ref: '#/definitions/fullName'
+                      type: 'object',
+                      properties: {
```

Comment:
The `fullName` definition also has `minLength` and `maxLength` properties for each of these keys, so this change will remove that restriction as well. Is this intended?

Reply:
Correct, anything that exceeds that maximum length of the 21-0781 form field will overflow to an additional page